reference/docs-conceptual/learn/ps101/07-working-with-wmi.md

```diff
@@ -1,68 +1,74 @@
 ---
+
 # Chapter 7 - Working with WMI
 
 ## WMI and CIM
 
-Windows PowerShell ships by default with cmdlets for working with other technologies such as Windows Management Instrumentation (WMI). The WMI cmdlets are deprecated and are not available in PowerShell 6+, but are covered here as you may encounter them in older scripts running on Windows PowerShell. For new development, use the CIM cmdlets instead.
+Windows PowerShell ships by default with cmdlets for working with other technologies, such as
+Windows Management Instrumentation (WMI). The WMI cmdlets are deprecated and aren't available in
+PowerShell 6+, but are covered here as you might encounter them in older scripts running on Windows
+PowerShell. For new development, use the CIM cmdlets instead.
 
-There are several native WMI cmdlets that exist in PowerShell without having to install any additional software or modules.  `Get-Command` can be used to
-determine what WMI cmdlets exist in Windows PowerShell. The following results are from my Windows 10 lab environment computer that is running PowerShell version 5.1. Your results may differ depending on what PowerShell version you're running. 
+Several native WMI cmdlets exist in PowerShell without you having to install any other software or
+modules. `Get-Command` can be used to determine what WMI cmdlets exist in Windows PowerShell. The
+following results are from a Windows 11 system running PowerShell version 5.1. Your results might
+differ depending on the PowerShell version you're running.
 
 ```powershell
 Get-Command -Noun WMI*
 ```
 
 ```Output
-CommandType     Name                                               Version    Source
------------     ----                                               -------    ------
-Cmdlet          Get-WmiObject                                      3.1.0.0    Microsof...
-Cmdlet          Invoke-WmiMethod                                   3.1.0.0    Microsof...
-Cmdlet          Register-WmiEvent                                  3.1.0.0    Microsof...
-Cmdlet          Remove-WmiObject                                   3.1.0.0    Microsof...
-Cmdlet          Set-WmiInstance                                    3.1.0.0    Microsof...
+CommandType     Name                                               Version
+-----------     ----                                               -------
+Cmdlet          Get-WmiObject                                      3.1.0.0
+Cmdlet          Invoke-WmiMethod                                   3.1.0.0
+Cmdlet          Register-WmiEvent                                  3.1.0.0
+Cmdlet          Remove-WmiObject                                   3.1.0.0
+Cmdlet          Set-WmiInstance                                    3.1.0.0
 ```
 
 Common Information Model (CIM) cmdlets were introduced in PowerShell version 3.0. The CIM cmdlets
-are designed so they can be used on both Windows and non-Windows machines. 
+are designed to be used on both Windows and non-Windows machines.
```

Comment:
CIM cmdlets only exist on Windows. They are not available on non-Windows.

Reply:
Updated.